dbtmetabase/parsers/dbt_manifest.py

```diff
@@ -0,0 +1,195 @@
+import json
+import os
+from typing import List, Iterable, Mapping, Optional, MutableMapping
+import logging
+
+from dbtmetabase.models.metabase import METABASE_META_FIELDS
+from dbtmetabase.models.metabase import MetabaseModel, MetabaseColumn
+
+
+class DbtManifestReader:
+    """
+    Reader for dbt manifest artifact.
+    """
+
+    def __init__(self, project_path: str):
+        """Constructor.
+
+        Arguments:
+            manifest_path {str} -- Path to dbt manifest.json.
+        """
+
+        self.manifest_path = os.path.expanduser(project_path)
+        self.manifest: Mapping = {}
+        self.catch_aliases: MutableMapping = {}
+
+    def read_models(
+        self,
+        database: str,
+        schema: str,
+        schemas_excludes: Iterable = None,
+        includes: Iterable = None,
+        excludes: Iterable = None,
+        include_tags: bool = True,
+        dbt_docs_url: str = None,
+    ) -> List[MetabaseModel]:
+
+        if schemas_excludes is None:
+            schemas_excludes = []
+        if includes is None:
+            includes = []
+        if excludes is None:
+            excludes = []
+
+        path = os.path.join(self.manifest_path)
```

Comment:
What are you joining here? Manifest path is already a string.

Reply:
removed os path join call